examples/tensor/static_tensor.cpp

```diff
@@ -32,7 +32,7 @@ int main() {
      * Value Type: float
      * Extents Type: dynamic_extents<5>{1,2,3,4,5}
     **/
-    auto t1 = ub::static_tensor<float,ub::static_extents<1,2,3,4,5> >{5.f};
+    auto t1 = ub::static_tensor<float,ub::static_extents<1,2,3,4,5> >{ub::static_extents<1,2,3,4,5>{}, 5.f};
```

Comment:
This is due to type aliasing. If I add a construction in `tensor_core` it conflicts with other constructors. I was thinking about how to solve it and try to avoid `std::enable_if`.

Reply:
Most probably I will solve it by next pull request.

Reply:
I would like to solve it with this pull request as @coder3101 and me would like to use as the lastest development.

Reply:
Yes, it will be really hard for me to work on if base continues to change significantly.

Reply:
Don't worry after this it will not change that much.